JeffShepherd/qwixx-clone
# Feature/cypress

## What does this PR do?

- Add cypress e2e testing
- Remove unused test folder and add two example tests
- Adds script to run cypress

## How should it be tested in the UI?

- Start application with 'npm start'
- Open another terminal tab and run 'npm run cypress' command to start cypress tests
- Open cypress in browser and run tests
- Both tests should pass
